<launch>

  <rosparam file="$(find dmp_motion_controller)/launch/r_arm_dmp_joint_position_controller.yaml" command="load"/>

  <rosparam ns="/r_arm_dmp_joint_position_controller" file="$(find dmp_motion_controller)/launch/r_arm_joint_position_gains.yaml" command="load"/>
  <rosparam ns="/r_arm_dmp_joint_position_controller" file="$(find dmp_motion_generation)/config/variable_names.yaml" command="load" />
  <rosparam ns="/r_arm_dmp_joint_position_controller" file="$(find dmp_motion_controller)/config/pr2_right_arm_joint_space_variable_names.yaml" command="load" />
  <rosparam ns="/r_arm_dmp_joint_position_controller" file="$(find dmp_motion_controller_test)/launch/dmp_test_params.yaml" command="load"/>

</launch>
